ceph.spec.in

```diff
@@ -492,6 +492,12 @@ export RPM_OPT_FLAGS=`echo $RPM_OPT_FLAGS | sed -e 's/i386/i486/'`
 		--prefix=/usr \
 		--localstatedir=/var \
 		--sysconfdir=/etc \
+%if 0%{?rhel} || 0%{?fedora}
+		--with-systemd-libexec-dir=/usr/lib/systemd/system \
+%endif
+%if 0%{?opensuse} || 0%{?suse_version}
+		--with-systemd-unit-dir=%_unitdir \
+%endif
```

Comment:
This section can be removed now; `pkg-config` will do the right thing for us on each platform.